### PR TITLE
[pool automation] Molecule docker driver

### PR DESCRIPTION
**Pull request by andkononykhin**
Signed-off-by: Andrey Kononykhin <andkononykhin@gmail.com>

### CODE REVIEW

File: pool_automation/roles/node_install/molecule/docker-ubuntu1604/molecule.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+platforms:
+  - name: instance
+    image: solita/ubuntu-systemd:latest
+    command: /sbin/init
+    capabilities:
+      - SYS_ADMIN
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    network_mode: ${NETWORK_MODE:-bridge}
```
Comment (skhoroshavin) [inline pool_automation/roles/node_install/molecule/docker-ubuntu1604/molecule.yml]: Is custom `network_mode` really needed here?

File: pool_automation/roles/node_install/molecule/docker-ubuntu1604/molecule.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    config-file: ../common/yamllint
+platforms:
+  - name: instance
+    image: solita/ubuntu-systemd:latest
+    command: /sbin/init
+    capabilities:
+      - SYS_ADMIN
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
```
Comment (skhoroshavin) [inline pool_automation/roles/node_install/molecule/docker-ubuntu1604/molecule.yml]: `privileged` is superset of `SYS_ADMIN` capability so one of them could be dropped. Also it seems like `SYS_ADMIN` is enough, so it's better to prefer `privileged`: https://molecule.readthedocs.io/en/latest/examples.html#systemd-container

File: pool_automation/roles/node_install/molecule/docker-ubuntu1604/molecule.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+---
```
Comment (skhoroshavin) [inline pool_automation/roles/node_install/molecule/docker-ubuntu1604/molecule.yml]: Given that vagrant is so much slower than docker and cannot be run inside VMs it think that making docker scenario default would be better

File: pool_automation/roles/node_install/molecule/docker-ubuntu1604/molecule.yml (hunk 1 above)
Comment (andkononykhin) [reply pool_automation/roles/node_install/molecule/docker-ubuntu1604/molecule.yml]: Yes, necessary in some circumstances (at least in my dev environment). Will think about that more.

File: pool_automation/roles/node_install/molecule/docker-ubuntu1604/molecule.yml (hunk 2 above)
Comment (andkononykhin) [reply pool_automation/roles/node_install/molecule/docker-ubuntu1604/molecule.yml]: ok

File: pool_automation/roles/node_install/molecule/docker-ubuntu1604/molecule.yml (hunk 3 above)
Comment (andkononykhin) [reply pool_automation/roles/node_install/molecule/docker-ubuntu1604/molecule.yml]: Yes, makes sense

**Comment by andkononykhin** [comment]
test this please

**Pull request merged**